context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FilePairSystemMonitor.java

```diff
@@ -92,68 +70,6 @@ public FilePairSystemMonitor(final DataSource dataSource,
         this.pathToMonitor = new File(this.dataSource.getUri());
```

Comment:
The logging statement should be moved to `AbstractFileSystemMonitor`'s constructor.

`this.pathToMonitor = new File(this.dataSource.getUri());` can be removed because it is already done in `AbstractFileSystemMonitor`'s constructor.

Reply:
[x] done